src/app/projects/states/dashboard/directives/task-dashboard/directives/task-status-card/task-status-card.coffee

```diff
@@ -38,10 +39,7 @@ angular.module('doubtfire.projects.states.dashboard.directives.task-dashboard.di
     # Triggers a new task status
     $scope.triggerTransition = (trigger) ->
       if trigger == 'ready_to_mark' && $scope.task.isPastTargetDate() && !$scope.task.isOverdue()
-        ConfirmationModal.show(
-          "Request Extensions?",
-          "This task is past the target due date, so you will need to request an extension to get feedback. Use confirm to request extensions and proceed with the submission.",
-          () -> extendAndSubmit(trigger))
+        ExtensionModal.show($scope.task)
```

Comment:
They need to be able to submit here - and then include the extension within the upload process. The backend has been changed to review the task status on extension - time exceeded with be updated to ready to mark if there is an extension.

Reply:
TriggerState is now passed as a callback to ExtensionModal.show